build-tools/automation/azure-pipelines.yaml

```
@@ -178,27 +178,29 @@ stages:
         artifactName: $(InstallerArtifactName)
         targetPath: xamarin-android/bin/Build$(XA.Build.Configuration)/$(InstallerArtifactName)
 
-    - task: DotNetCoreCLI@2
-      displayName: pack Xamarin.Android.Sdk
+    - task: MSBuild@1
+      displayName: pack runtime nupkgs
       inputs:
-        command: pack
-        packagesToPack: $(System.DefaultWorkingDirectory)/xamarin-android/src/Xamarin.Android.Sdk/Xamarin.Android.Sdk.Lite.proj
+        solution: $(System.DefaultWorkingDirectory)/xamarin-android/src/Xamarin.Android.App/Xamarin.Android.App.Runtime.proj
         configuration: $(XA.Build.Configuration)
-        packDirectory: $(System.DefaultWorkingDirectory)/xamarin-android/bin/Build$(XA.Build.Configuration)/$(NuGetArtifactName)
+        msbuildArguments: /t:_PackAllRuntimePacks /restore /bl:$(System.DefaultWorkingDirectory)/bin/Build$(XA.Build.Configuration)/create-runtime-packs.binlog
 
     - task: DotNetCoreCLI@2
-      displayName: pack Xamarin.Android.Sdk.Lite
+      displayName: pack sdk and ref nupkgs
       inputs:
         command: pack
-        packagesToPack: $(System.DefaultWorkingDirectory)/xamarin-android/build-tools/create-packs/Packager.csproj
+        packagesToPack: "\
+          $(System.DefaultWorkingDirectory)/xamarin-android/src/Xamarin.Android.App/Xamarin.Android.App.Ref.proj;
+          $(System.DefaultWorkingDirectory)/xamarin-android/src/Xamarin.Android.Sdk/Xamarin.Android.Sdk.proj;
+          $(System.DefaultWorkingDirectory)/xamarin-android/src/Xamarin.Android.Sdk/Xamarin.Android.Sdk.Lite.proj"
```

Can this use the regular multi-line syntax?
```suggestion
        packagesToPack: |
          $(System.DefaultWorkingDirectory)/xamarin-android/src/Xamarin.Android.App/Xamarin.Android.App.Ref.proj;
          $(System.DefaultWorkingDirectory)/xamarin-android/src/Xamarin.Android.Sdk/Xamarin.Android.Sdk.proj;
          $(System.DefaultWorkingDirectory)/xamarin-android/src/Xamarin.Android.Sdk/Xamarin.Android.Sdk.Lite.proj
```
Yaml is weird, maybe it doesn't work.

---

`|` will keep new lines, and `>` will replace new lines with spaces. In this case the dotnet CLI yaml task expects `;` as the delimiter. This seemed like the only way to pass multiple projects on separate lines.